advanced/Scripts/COL_TABLE

```diff
@@ -1,20 +1,40 @@
-COL_NC='\e[0m' # No Color
-COL_WHITE='\e[1;37m'
-COL_BLACK='\e[0;30m'
-COL_BLUE='\e[0;34m'
-COL_LIGHT_BLUE='\e[1;34m'
-COL_GREEN='\e[0;32m'
-COL_LIGHT_GREEN='\e[1;32m'
-COL_CYAN='\e[0;36m'
-COL_LIGHT_CYAN='\e[1;36m'
-COL_RED='\e[0;31m'
-COL_LIGHT_RED='\e[1;31m'
-COL_PURPLE='\e[0;35m'
-COL_LIGHT_PURPLE='\e[1;35m'
-COL_BROWN='\e[0;33m'
-COL_YELLOW='\e[1;33m'
-COL_GRAY='\e[0;30m'
-COL_LIGHT_GRAY='\e[0;37m'
+if [[ -t 1 ]] && [[ $(tput colors) -ge 8 ]]; then
+  COL_NC='\e[0m'
+  COL_WHITE='\e[1;37m'
+  COL_BLACK='\e[0;30m'
+  COL_BLUE='\e[0;34m'
+  COL_LIGHT_BLUE='\e[1;34m'
+  COL_GREEN='\e[0;32m'
+  COL_LIGHT_GREEN='\e[1;32m'
+  COL_CYAN='\e[0;36m'
+  COL_LIGHT_CYAN='\e[1;36m'
+  COL_RED='\e[0;31m'
+  COL_LIGHT_RED='\e[1;31m'
+  COL_PURPLE='\e[0;35m'
+  COL_LIGHT_PURPLE='\e[1;35m'
+  COL_BROWN='\e[0;33m'
+  COL_YELLOW='\e[1;33m'
+  COL_GRAY='\e[0;30m'
+  COL_LIGHT_GRAY='\e[0;37m'
+else
```

Comment:
I'm wondering about this, I think we can just get away with the first part of the if block, don't think we need the else part!